common/network-common/src/main/java/org/apache/spark/network/client/TransportResponseHandler.java

```diff
@@ -188,6 +188,7 @@ public void handle(ResponseMessage message) throws Exception {
       if (listener == null) {
         logger.warn("Ignoring response for RPC {} from {} ({} bytes) since it is not outstanding",
           resp.requestId, getRemoteAddress(channel), resp.body().size());
+        resp.body().release();
```

Comment:
From the commit history, is this missed since Apache Spark 1.2.0 at SPARK-3453?

Reply:
Evidently, yeah. It may just not cause much of an issue. Are there any reasons to release() it, anyone know?